README.md

```diff
@@ -9,3 +9,27 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Answer 
+Direct Proof:
+- If we have two graphs $A$ and $B$ that are both completely connected and have the same number of nodes.
+  - $A$ has a set of vertices $V_{A}$ and a set of edges $E_{A}$.
+  - $B$ has a set of vertices $V_{B}$ and a set of edges $E_{B}$.
+  - $|V_{A}| = m$ and $|V_{B}| = n$ where $m = n$.
+  
+- Since $|V_{A}| = |V_{B}|$ the two graphs have the same cardinality (the same number of elements). According to theorem 2 of the pigeonhole principle "If X and Y have the same number of elements and f is one-to-one, then f is onto". This means there exists a bijection function
```

Comment:
You're still not telling me what $f$ is -- you can't just assume that it exists.

Reply:
Ok, I think I was focusing on not using an example. But over looking that I need to show that the function can be constructed still? So I am thinking that I a need to show that generally if we have a function $f(v_{1})$ from the graph $V_{A}$ it is equal to the corresponding vertex in $V_{b}$. Is this on the write track of what you are hinting at?

Reply:
Yes -- the fundamental question is how you find the corresponding vertex.